lib/node_modules/@stdlib/stats/base/snanmeanpn/docs/repl.txt

```diff
@@ -89,10 +86,8 @@
 
     // Using offset parameter:
     > var x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 3.0, 2.0, 5.0, -1.0, NaN ] );
-    > var N = {{alias:@stdlib/math/base/special/floor}}( x.length / 2 );
-    > {{alias}}.ndarray( N, x, 2, 1 )
+    > {{alias}}.ndarray( 3, x, 2, 1 )
     ~-0.3333
 
     See Also
     --------
```

Comment:
```suggestion
    --------

```

No reason to delete this empty line. While not wholly consistent, we typically include in our repl help files to ensure proper spacing in the REPL.